crypto/pem/pem_lib.c

```diff
@@ -44,28 +44,19 @@ int PEM_def_callback(char *buf, int num, int w, void *key)
     if (prompt == NULL)
         prompt = "Enter PEM pass phrase:";
 
-    for (;;) {
-        /*
-         * We assume that w == 0 means decryption,
-         * while w == 1 means encryption
-         */
-        int min_len = w ? MIN_LENGTH : 0;
+    /*
+     * We assume that w == 0 means decryption,
+     * while w == 1 means encryption
+     */
```

Comment:
I wonder if it would be appropriate to use this opportunity to rename the variable to something more informative. At least in EVP_read_pw_string_min declaration it's called "verify", and it's so much better than "w". Naturally some commentary clarifying would be appropriate too. Like "verify == 1 is used with encryption"...

Reply:
Good idea